src/export.js

```diff
@@ -14,13 +14,18 @@
  */
 
 const { Gtk, Gio } = imports.gi
+const GLib = imports.gi.GLib
 const ngettext = imports.gettext.ngettext
 const { mimetypes, readJSON, sepHeaderFunc } = imports.utils
 const { EpubViewAnnotation } = imports.epubView
 const { EpubCFI } = imports.epubcfi
 const { AnnotationRow } = imports.contents
 
-const exportToHTML = async ({ annotations }, metadata, getSection) => {
+const exportToHTML = async (jsonData, metadata, getSection, withJson) => {
+    const json_base64 = withJson ? 
+        `<script id="json-db" type="application/json">${GLib.base64_encode(JSON.stringify(jsonData))}</script>` : ''      
```

Comment:
You don't have to base64 encode the JSON if you put it in `<script>`; `JSON.stringify()` should be enough.

Reply:
The idea was to avoid the `<br>` breaking the code. This practice seems to be standard.

Reply:
`<br>` won't break anything. `</script>` will. So one should just escape `</script>`, rather than base64 encoding.

Besides, if base64 is used, one should probably do something like
```
<script type="application/json">
{
   data: "${base64 string here}"
}
</script>
```

Because a naked Base64 string is not valid JSON.

Reply:
The Base64 gives some data integrity.
How would you escape `</script>` ?